### ISSUE TITLE
Login page will not load on protected wifi networks after update: N2G48B.2017.07.06.00.04.39 release

### ISSUE THREAD

**Issue by theGrower** [copperhead/bugtracker#659]
The login URL will not load after connecting to a protected network (provider's login webpage). I believe this is the CaptivePortalLogin APK?

**Comment by thestinger** [comment]
We'll see if this is fixed by the next release disabling the WiFi toggle permission review code. It may also be a compatibility issue with the WebView being provided by Monochrome, requiring some kind of change to how that's done to match how Google does it.

**Comment by thestinger** [comment]
Please upgrade to N2G48B.2017.07.06.18.26.24 and try there.

**Comment by theGrower** [comment]
Still the same outcome. I'll have to get logs later.

**Comment by thestinger** [comment]
@theGrower Does Chromium work?

**Comment by theGrower** [comment]
Actually, I'm sorry. I won't be able to test this until tomorrow. The toggle is working, so I'll assume this will too. However the other bugs were not fixed. Plumbing issues kept me from getting logs. Tomorrow it will have to be.

**Comment by thestinger** [comment]
Knowing if the included Chromium app works for you is a higher priority than the logs.

**Comment by theGrower** [comment]
I did have Chromium disabled, and have since the day (several months ago) I flashed the OS. This has never been a problem previously. 
I have been unable to retest this as the login at work is good for 12 hours. The workaround is to disable mobile data, then pull up the company website in a browser to get it to redirect to the log in page. It just won't go to the page through the CaptivePortalLogin. 
I was also unable to get logs as I do not have ADB at work, and IT says that is a "no no."
